src/js/tasks/tasks.js

```diff
@@ -719,15 +719,38 @@ subtask("setPrices", "Update Lido ARM's swap prices")
     undefined,
     types.boolean
   )
+  .addOptionalParam(
+    "priceOffset",
+    "Offset the 1Inch/Curve price by this amount in basis points (above for buy, below for sell, reversed if negative). eg 100 = 1%",
+    0,
+    types.float
+  )
   .setAction(async (taskArgs) => {
     const signer = await getSigner();
 
     const armAddress = await parseDeployedAddress(
       `${taskArgs.arm.toUpperCase()}_ARM`
     );
-    const armContract = await ethers.getContractAt("AbstractARM", armAddress);
+    const arm = await ethers.getContractAt([
+      "function traderate0() external view returns (uint256)",
+      "function traderate1() external view returns (uint256)",
+      "function crossPrice() external view returns (uint256)",
+      "function activeMarket() external view returns (address)",
+      "function setPrices(uint256,uint256) external",
+    ], armAddress);
+
+    const activeMarket = "0x9a8bC3B04b7f3D87cfC09ba407dCED575f2d61D8"; //await arm.activeMarket();
+    if (activeMarket === ethers.ZeroAddress) {
+      log("No active lending market found, using default APY of 0%");
```

Comment:
change this to a console.log as there is no logger in this file.

Reply:
Indeed, fixed in this commit: [7e19215](https://github.com/OriginProtocol/arm-oeth/pull/111/commits/7e192159b5659ddbd8bab25e93b6e3acda11ce8e)